XamarinCommunityToolkit/Views/AvatarView/ImageSourceValidator.android.ios.macos.uwp.wpf.gtk.tizen.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+#if MONOANDROID
+using Xamarin.Forms.Platform.Android;
+#elif __IOS__
+using Xamarin.Forms.Platform.iOS;
+#elif __MACOS__
+using Xamarin.Forms.Platform.MacOS;
+#elif UWP
+using Xamarin.Forms.Platform.UWP;
+#elif NET471
+using Xamarin.Forms.Platform.GTK.Renderers;
+#elif TIZEN
+using Xamarin.Forms.Platform.Tizen;
+#else
+using Xamarin.Forms.Platform.WPF;
+#endif
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+#if MONOANDROID || __IOS__ || __MACOS__
+	using UriImageSourceHandler = ImageLoaderSourceHandler;
+#endif
+
+#if MONOANDROID || __IOS__ || __MACOS__ || NET471
+	using StreamImageSourceHandler = StreamImagesourceHandler;
+#endif
+
+	class ImageSourceValidator : IImageSourceValidator
+	{
+		public async Task<bool> IsImageSourceValidAsync(ImageSource source)
+		{
+			var handler = GetHandler(source);
+			if (handler == null)
+    			return true;
+
+#if TIZEN
+			return await handler.LoadImageAsync(null, source).ConfigureAwait(false);
+#elif MONOANDROID
+			var imageSource = await handler.LoadImageAsync(source, null);
```

Comment:
Tested here and works fine.
```suggestion
			var imageSource = await handler.LoadImageAsync(source, null).ConfigureAwait(false);
```